models/User.js

```diff
@@ -0,0 +1,17 @@
+export default (sequelize, DataTypes) => {
+  const User = sequelize.define(
+    'User',
```

Comment:
This is how the model should be structured for purposes of work efficiency as the teammates have concluded
<img width="1440" alt="Screen Shot 2019-07-09 at 09 01 37" src="https://user-images.githubusercontent.com/41841554/60866277-3601f400-a228-11e9-82ba-f066141ada71.png">